### PR TITLE
Modernize website

### PR DESCRIPTION
**Pull request by dmitry-vlass**
Websites like that cannot grow without the modern framework and great logo.

declan

### CODE REVIEW

**Comment by dmitry-vlass** [comment]
:rocket:[canva-check-merge]

```ini
CANVA_CHECK_MERGE_YOLO=true
```

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (declanvong) [inline public/index.html]:
```suggestion
    <title>Hey Declan</title>
```

File: public/manifest.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
```
Comment (declanvong) [inline public/manifest.json]: Please update manifest

File: src/logo.svg [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 841.9 595.3">
```
Comment (declanvong) [inline src/logo.svg]: Unused, please remove

File: tsconfig.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
+{
+  "compilerOptions": {
+    "target": "es5",
```
Comment (declanvong) [inline tsconfig.json]: do we live in 2009? es6 or nothing

File: public/index.html [hunk 5]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
```
Comment (declanvong) [inline public/index.html]: We cant let people know we used a script to create this website, that is low quality

**Comment by dmitry-vlass** [comment]
Thanks for the review, @declanvong. It was legendary.

**Comment by joshuaobrien** [comment]
:rocket:[canva-check-merge]

```ini
CANVA_CHECK_MERGE_SKIP_TRANSLATION=true
```

**Comment by dmitry-vlass** [comment]
:rocket:[canva-check-merge]

```ini
CANVA_CHECK_MERGE_YOLO=true
```

**Comment by joshuaobrien** [comment]
:rocket:[some-company-check-merge]

```ini
SOME_COMPANY_CHECK_MERGE_SKIP_TRANSLATION=true
```

**Pull request merged**